src/init.cpp

```diff
@@ -765,9 +765,9 @@ static std::string ResolveErrMsg(const char * const optname, const std::string&
     return strprintf(_("Cannot resolve -%s address: '%s'"), optname, strBind);
 }
 
-static std::string AmountErrMsg(const char * const optname, const std::string& strValue)
+static std::string AmountErrMsg(const std::string& optname, const std::string& strValue)
 {
-    return strprintf(_("Invalid amount for -%s=<amount>: '%s'"), optname, strValue);
+    return strprintf(_("Invalid amount for %s."), strprintf("-%s (%s=%s)", optname, optname, strValue));
```

Comment:
I don't think you can ask someone to "translate" a collection of characters (`-%s=<amount>: '%s'`) into a language.

Also, this seems to work with other grammar just fine:

![screenshot from 2016-01-19 08-43-51](https://cloud.githubusercontent.com/assets/6399679/12412671/d6519866-be88-11e5-88e6-a5199306cbaa.png)


Am I missing something?